src/doc/rustc/src/platform-support/x86_64_unknown_redox.md

```diff
@@ -0,0 +1,29 @@
+# `x86_64-unknown-redox`
+
+**Tier: 2**
+
+x86_64 Redox OS
+
+## Target maintainers
+
+- Missing!
```

Comment:
cc @jackpot51

Reply:
I would be maintaining any Redox platforms.